tests/test_cli_facade_behavior_parity.py

```
@@ -0,0 +1,100 @@
+ArgumentParser itself rejects, an exit code the facade fails to propagate)
+passes it silently. Each case below is a repro that regressed exactly that
+way during the facade migration and is pinned here so it cannot regress
+again.
+"""
+from __future__ import annotations
+
+import os
+import subprocess
+import sys
+
+SOCKET_PATH = "/tmp/cmux-facade-behavior-parity-absent.sock"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-facade-behavior-parity-absent.sock"

(S108)